packages/graphql/src/translate/queryAST/ast/filters/RelationshipFilter.ts

```diff
@@ -25,10 +25,10 @@ import { QueryASTContext } from "../QueryASTContext";
 import type { RelationshipAdapter } from "../../../../schema-model/relationship/model-adapters/RelationshipAdapter";
 
 export class RelationshipFilter extends Filter {
-    private targetNodeFilters: Filter[] = [];
-    private relationship: RelationshipAdapter;
-    private operator: RelationshipWhereOperator;
-    private isNot: boolean;
+    protected targetNodeFilters: Filter[] = [];
```

Comment:
On the ConnectionFilter this property is named `innerFilters` not that it's a better name but we could choose a consistent name for both!

Reply:
Agree, I'll review all filters holistically to find common naming